runtime/doc/eval.txt

```diff
@@ -9415,6 +9415,7 @@ tgetent			Compiled with tgetent support, able to use a termcap
 timers			Compiled with |timer_start()| support.
 title			Compiled with window title support |'title'|.
 toolbar			Compiled with support for |gui-toolbar|.
+touchbar		Compiled with support for Touch Bar in MacVim.
```

Comment:
I'm actually not 100% if we should expose a feature test for this, but I guess if toolbar is already one, why not.

Reply:
thank you that you've exposed it as Tool bar and Touch Bar are different. One can be enabled without other